consensus/src/pipeline/execution_schedule_phase.rs

```diff
@@ -61,12 +61,12 @@ impl StatelessPipeline for ExecutionSchedulePhase {
 
     async fn process(&self, req: ExecutionRequest) -> ExecutionWaitRequest {
         let ExecutionRequest {
-            ordered_blocks,
+            mut ordered_blocks,
             lifetime_guard,
         } = req;
 
-        let block_id = match ordered_blocks.last() {
-            Some(block) => block.id(),
+        let (block_id, pipeline_enabled) = match ordered_blocks.last() {
+            Some(block) => (block.id(), block.pipeline_enabled()),
```

Comment:
why is the flag carried on the block again?

Reply:
to determine whether it uses the new pipeline or not?